src/core/egg-herbie.rkt

```diff
@@ -473,7 +503,7 @@
         (for ([rule (in-list rules)])
           (define egg&ffi-rules
             (hash-ref! (*egg-rule-cache*)
-                       (cons (*active-platform*) rule)
+                       rule
```

Comment:
I guess because the active platform can't change for a single benchmark, and it's reset between benchmarks?

Reply:
Yep that was my thought.